backend/src/main/java/meetup/connect/meetupevent/GoogleCalendar.java

```diff
@@ -0,0 +1,204 @@
+        try {
+            service = getService();
+        } catch (GeneralSecurityException | IOException e) {
+            throw new RuntimeException(e);
+        }
+        Event event = fromMeetUpEvent(meetUpEvent);
+
+        try {
+            Event createdEvent = service.events().insert(CALENDAR_ID, event).execute();
+            return createdEvent.getId();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void deleteEvent(String eventId) {
+        try {
+            service = getService();
+        } catch (GeneralSecurityException | IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        try {
+            service.events().delete(CALENDAR_ID, eventId).execute();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void addAttendee(String attendeeEmail, String eventId) {
+        try {
+            service = getService();
+        } catch (GeneralSecurityException | IOException e) {
+            throw new RuntimeException(e);
+        }
+        try {
+
+            Event event = service.events().get(CALENDAR_ID, eventId).execute();
+
+            List<EventAttendee> attendees =
+                    CollectionUtils.isEmpty(event.getAttendees()) ? new ArrayList<>() : event.getAttendees();
+            attendees.add(new EventAttendee().setEmail(attendeeEmail));
+            event.setAttendees(attendees);
+
+            service.events().update(CALENDAR_ID, event.getId(), event).execute();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void removeAttendee(String attendeeEmail, String eventId) {
+        try {
+            service = getService();
+        } catch (GeneralSecurityException | IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        try {
+
+            Event event = service.events().get(CALENDAR_ID, eventId).execute();
+
+            List<EventAttendee> attendees =
+                    CollectionUtils.isEmpty(event.getAttendees()) ? new ArrayList<>() : event.getAttendees();
+            Optional<EventAttendee> firstAttendeeWithEmail =
+                    attendees.stream()
+                            .filter(attendee -> attendeeEmail.equals(attendee.getEmail()))
+                            .findFirst();
+            if (firstAttendeeWithEmail.isPresent()) {
+                EventAttendee attendee = firstAttendeeWithEmail.get();
+                attendees.remove(attendee);
+            } else {
+                log.error("Attendee does not exists");
+            }
+            event.setAttendees(attendees);
+
+            service.events().update(CALENDAR_ID, event.getId(), event).execute();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static Event fromMeetUpEvent(MeetUpEvent meetUpEvent) {
```

Comment:
note: logic looks ok, however, I wouldn't include business logic in the `GoogleCalendar` class. The class should be only responsible for performing calendar operations. You violated the single responsibility principal.

Reply:
move that logic to the `MeetUpEventService` class